### DISCUSSION TITLE
No such file or directory @ dir_chdir - /github/workspace/docs

### DISCUSSION THREAD

**Discussion by w-decker** [community/community#61572] [Actions] [answered]
### Select Topic Area

Question

### Body

I am trying to deploy a repo to GitHub pages using Jupyter Cookiecutter. After many tries, I have been unusccessful in deploying my site. I keep receiving this error:

github-pages 228 | Error:  No such file or directory @ dir_chdir - /github/workspace/docs

After many attempts I have not been succesful in debugging this error. I have found similar discussion but the answers did not seem to concretely point me to a solution to the problem. Any help is appreciated. The repo is: https://github.com/w-decker/hidden-markov-model.

**Comment by omar-chaabouni** [comment]
got any solution ? please share, thanks

**Comment by imanrd** [comment, marked as answer]
My problem was solved with this comment: https://github.com/orgs/community/discussions/76938#discussioncomment-7665841
